webapp/src/components/modals/configModal.tsx

```diff
@@ -0,0 +1,739 @@
+import React, {useEffect, useState} from 'react';
+
+import Button from 'react-bootstrap/Button';
+import Modal from 'react-bootstrap/Modal';
+import Form from 'react-bootstrap/Form';
+import Table from 'react-bootstrap/Table';
+import ButtonGroup from 'react-bootstrap/ButtonGroup';
+import {OverlayTrigger, Tooltip, ToggleButton} from 'react-bootstrap';
```

Comment:
Check if these can be combined in a single import

Reply:
it is not working in single import